### PR TITLE
UX: More improvements to login/signup forms

### PR DESCRIPTION
**Pull request by jancernik**
This PR adds more improvements to the signup/login forms. The main ones are:

- Validations for email and password will appear for the first time only after the input loses focus.
- Form inputs will scroll into view when they gain focus.
- Borders around the content on the account-created/activation pages are removed.
- Plugin outlets are hidden for fullpage login/signup.
- A plain background is used in the fullpage forms.
- The Font size of the login/signup CTA on mobile is bigger.
- Alerts are closer to the form in fullpage.

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/modal/create-account.js [hunk 1]
```diff
@@ -215,8 +217,23 @@ export default class CreateAccount extends Component.extend(
     });
   }
 
+  @action
+  showPasswordValidation() {
+    if (this.passwordValidation.reason) {
+      this.set("passwordValidationVisible", true);
+    } else {
+      this.set("passwordValidationVisible", false);
+    }
```
Comment (pmusaraj) [inline app/assets/javascripts/discourse/app/components/modal/create-account.js:226]: code ⛳ you could also do: 

```suggestion
    this.set("passwordValidationVisible", Boolean(this.passwordValidation.reason));
```

File: app/assets/javascripts/discourse/app/components/modal/create-account.js [hunk 2]
```diff
@@ -215,8 +217,23 @@ export default class CreateAccount extends Component.extend(
...
+  }
+
   @action
   checkEmailAvailability() {
+    if (this.emailValidation.reason) {
+      this.set("emailValidationVisible", true);
+    } else {
+      this.set("emailValidationVisible", false);
+    }
+
```
Comment (pmusaraj) [inline app/assets/javascripts/discourse/app/components/modal/create-account.js:236]: same `Boolean` trick here too

File: app/assets/stylesheets/common/base/activation.scss [hunk 3]
```diff
@@ -9,6 +9,9 @@
     padding: 0;
     height: 100%;
   }
+  .above-main-container-outlet {
+    display: none;
+  }
```
Comment (pmusaraj) [inline app/assets/stylesheets/common/base/activation.scss:14]: There's no easy way to remove the outlet in JS, right?

**Review by pmusaraj — APPROVED** [review]
Looks good overall, just a few minor comments.

File: app/assets/javascripts/discourse/app/components/modal/create-account.js (hunk 1 above)
Comment (jancernik) [reply app/assets/javascripts/discourse/app/components/modal/create-account.js]: oc, nice! :ok_hand:

File: app/assets/stylesheets/common/base/activation.scss (hunk 3 above)
Comment (jancernik) [reply app/assets/stylesheets/common/base/activation.scss:14]: Not really as it lives in `application.hbs`

https://github.com/discourse/discourse/blob/852ff18fb94f09c8fea2a2bd9f9277a5c27f4c51/app/assets/javascripts/discourse/app/templates/application.hbs#L53

**Pull request merged**